recipes/cocotb/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cocotb" %}
+{% set version = "1.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cocotb-{{ version }}.tar.gz
+  sha256: 1f390bb6468d70e04acebeb9ff05b54be42cbbbbbb3bd6c75d1ec0a36cb97e23
+
+build:
+  noarch: python
```

Comment:
This package is not noarch. [It contains compiled code.](https://github.com/cocotb/cocotb/blob/v1.7.2/setup.py#L90)

```suggestion
```